index.js

```diff
@@ -0,0 +1,140 @@
+// =====xxxxx===== first question:
+// Check if two given integer values are in the range 50 to 99 (this includes both 50 and 99). Print true if either of them are in the range.
+console.log("first question:");
+
+let x = 20;
+let y = 29;
+ 
+if (x >= 50 && x <= 99 , y >= 50 && y <= 99) {
```

Comment:
you so close ==> we can not use ',' in if statment => but think about logical oparators like & |